containers/cuda/Containerfile

```diff
@@ -14,10 +14,7 @@ RUN export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/cuda/lib64:/usr/local/cu
     && export XLA_TARGET=cuda120 \
     && export XLA_FLAGS=--xla_gpu_cuda_data_dir=/usr/local/cuda
 
-ARG GIT_TAG=stable
-RUN if [[ "$(uname -m)" != "aarch64" ]]; then export CFLAGS="-mno-avx"; fi && \
-    CMAKE_ARGS="-DLLAMA_CUBLAS=on" python3.11 -m pip install -r https://raw.githubusercontent.com/instructlab/instructlab/${GIT_TAG}/requirements.txt --force-reinstall --no-cache-dir llama-cpp-python
-RUN python3.11 -m pip install git+https://github.com/instructlab/instructlab.git@${GIT_TAG}
+RUN python3.11 -m pip install instructlab
```

Comment:
This will build llama-cpp-python without CUDA support and does not install InstructLab's CUDA dependencies.

Reply:
Hi @tiran, thanks for the reply. Given the PR @seroyer linked, does it make sense to pursue this? 

If so, would `CMAKE_ARGS="DLLAMA_CUBLAS=on" python3.11 -m pip install instructlab[cuda]` install CUDA dependencies?

Reply:
It's `-D` to set a cmake define.

Installation is a bit more complicated, because one dependencies (`flash-attn`) does not specify its build and installation requirements correctly. You have to install instructlab and two supports packages, then install instructlab's CUDA dependencies:

```
CMAKE_ARGS="-DLLAMA_CUDA=on -DLLAMA_NATIVE=off" python3.11 -m pip install instructlab packaging wheel \
   && python3.11 -m pip install instructlab[cuda]
```